plugins/pmsmart/src/test/java/org/itsallcode/whiterabbit/plugin/pmsmart/integration/Tester.java

```diff
@@ -0,0 +1,105 @@
+package org.itsallcode.whiterabbit.plugin.pmsmart.integration;
+
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.YearMonth;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.itsallcode.whiterabbit.api.PluginConfiguration;
+import org.itsallcode.whiterabbit.api.features.ProgressMonitor;
+import org.itsallcode.whiterabbit.api.model.DayType;
+import org.itsallcode.whiterabbit.api.model.ProjectReport;
+import org.itsallcode.whiterabbit.api.model.ProjectReportActivity;
+import org.itsallcode.whiterabbit.api.model.ProjectReportDay;
+import org.itsallcode.whiterabbit.logic.Config;
+import org.itsallcode.whiterabbit.logic.report.project.ProjectReportGenerator;
+import org.itsallcode.whiterabbit.logic.report.project.ProjectReportImpl;
+import org.itsallcode.whiterabbit.logic.report.project.ProjectReportImpl.DayImpl;
+import org.itsallcode.whiterabbit.logic.report.project.ProjectReportImpl.ProjectActivityImpl;
+import org.itsallcode.whiterabbit.logic.service.contract.ContractTermsService;
+import org.itsallcode.whiterabbit.logic.service.project.ProjectImpl;
+import org.itsallcode.whiterabbit.logic.service.project.ProjectService;
+import org.itsallcode.whiterabbit.logic.storage.CachingStorage;
+import org.itsallcode.whiterabbit.logic.storage.Storage;
+import org.itsallcode.whiterabbit.logic.storage.data.JsonFileStorage;
+import org.itsallcode.whiterabbit.plugin.pmsmart.PMSmartExporter;
+import org.itsallcode.whiterabbit.plugin.pmsmart.web.WebDriverFactory;
+
+/**
+ * Intended for manual tests
+ */
+public class Tester
+{
+    private static final String PROJECT_FILE = "c:/HOME/Tmp/White-Rabbit-Test-Data/projects.json";
+    private static final String TIME_RECORDING_FILE = "c:/HOME/Tmp/White-Rabbit-Test-Data/time-record.json";
```

Comment:
This depends on a local file. Either include it as a resource or define test data within the class.

Reply:
I felt (and interpreted) the intention of this class to enable manual real world tests with actual user data.

A sample use case was based on issue #151:
* Strange behavior 
* hard to reproduce
* suspected sporadic failures 
 
When observing this I wanted to create a framework in order to test with real data as easy as possible. 
So a developer can easily feed in his real files and investigate the observed issue.

Creating all these data in the class seemed to be inefficient and future issues might require the same process again. 
Additional test cases did not seem to be not feasible either, since the behavior reported by issue #151 is an integration issue.

Would it be better to replace the paths by fake paths like "/path/to/your/time-recording/for/year-month.json"?

If this still conflicts with code policy then I'd rather prefer to remove file Tester.java and keep it outside the repo.
This is how I found it when starting investigation.

What do you suggest?

Reply:
OK, now I understand your intention, and that's fine for me. Please add a comment to the class and use dummy paths to make the intention clear :)
Another option would be to read an `.gitignore`d custom property file pointing to the real file. But in this case this is too much effort.

Reply:
Thanks - done.
Class comment had been provided already.
Great!